pkg/vault/test/foundry/E2eSwap.t.sol

```diff
@@ -798,6 +800,11 @@ contract E2eSwapTest is BaseVaultTest {
             );
         }
 
+        {
+            BaseVaultTest.Balances memory balancesMiddle = getBalances(sender, Rounding.ROUND_DOWN);
```

Comment:
Let's add a small comment here explaining why we need this (fees / need enough tokens to do the roundrip, etc)